File: asp-net-core/JwtAuthApp/Domain/DTO/DataDTO.cs
using System.Collections.ObjectModel;

namespace JwtAuthApp.Domain.DTO;

public class DataDTO
{
    public string Name { get; set; }
    public string Surname { get; set; }
    public string PhoneNumber { get; set; }
    public int IntegerNumber { get; set; }
    public double DecimalNumber { get; set; }
    public Collection<AddressDTO> Addresses { get; set; }
    public Collection<MemberDTO> Members { get; set; }

}

File: spring/jwt-token-auth-demo/src/main/java/com/sggw/jwttokenauthdemo/dto/DataDTO.java
package com.sggw.jwttokenauthdemo.dto;

import java.util.List;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Pattern;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DataDTO {
    
    private String name;

    private String surname;

    @NotBlank(message = "Invalid Phone number: Empty number")
    @NotNull(message = "Invalid Phone number: Number is NULL")
    @Pattern(regexp = "^\\d{10}$", message = "Invalid phone number")
    private String phoneNumber;

    private int integerNumber;

    private double decimalNumber;

    private List<AddressDTO> addresses;

    private List<MemberDTO> members;


}
